### PR TITLE
Dark theme and search

### PR DESCRIPTION
**Pull request by ApurvaSawant11**
In this PR, I have added Dark theme to the project and Search feature.

Please review and suggest

### CODE REVIEW

File: src/pages/landing/landing.css [hunk 1]
```diff
@@ -3,7 +3,6 @@
     height: 100%;
```
Comment (MayankKumar10) [inline src/pages/landing/landing.css:3]: by default, height is 100%

File: src/components/searchBar/searchBar.css [hunk 2]
```diff
@@ -19,17 +23,55 @@
...
+}
+
+/* || Media Queries */
+
+@media screen and (max-width: 1080px) {
+    .search-container {
+        width: 20rem;
+    }
+    .modal {
+        margin-left: 40% ;
```
Comment (MayankKumar10) [inline src/components/searchBar/searchBar.css:46]: Don't use large value for margins instead try to use flex or grid for that

**Review by MayankKumar10 — APPROVED** [review]
Look into these Suggestions Everything else looks good to me you can merge it

File: src/components/header/header.css [hunk 3]
```diff
@@ -1,17 +1,67 @@
...
+        padding-left: 0.7rem;
+    }
+
+    header {
+        height: 4.4rem;
+    }
+
+    .hamburger-icon {
+        margin: 0;
+    }
```
Comment (Dev-AkshatGupta) [inline src/components/header/header.css:65]: Can make such utility classes instead

**Review by Dev-AkshatGupta — APPROVED** [review]
Please consider suggestion everything else seems good to me

File: src/components/header/header.css (hunk 3 above)
Comment (ApurvaSawant11) [reply src/components/header/header.css:65]: I  have used utility classes. This is in media queries for responsiveness

File: src/components/searchBar/searchBar.css (hunk 2 above)
Comment (ApurvaSawant11) [reply src/components/searchBar/searchBar.css:46]: Okay. Will fix this in next PR

**Pull request merged**